### ISSUE TITLE
WISH: Pick on multiple target

### ISSUE THREAD

**Issue by GiuseppeChillemi** [red/REP#120]
Actually you can pick just one value but in some use cases, having pick return multiple data would be good. 
Table made with block of blocks is one of the most important one:

```
table: [[a b c d] [x y z w] [5 6 7 8]]
pick second table [1 3]
>>  [x z]
```

**Comment by hiiamboris** [comment]
I think your result should be `== [[x y z w] none]`

**Comment by GiuseppeChillemi** [comment]
@hiiamboris Thank you, I have copied it from Gitter where there was the error. Now it should be correct

**Comment by endo64** [comment]
@GiuseppeChillemi This can be done very easily on mezz level, in this sense not everything should be added to the language as a feature:

    t: [x y z w]
    pick*: func [t b] [collect [forall b [keep pick t b/1]]]
    pick* t [1 3]
    ; == [x z]

    pick* t [1 9]
    ; == [x none]

**Comment by GiuseppeChillemi** [comment]
@endo64 You are right and I have already it at mezzanine level. I have requested this, because `pick` is a frequent use function in strict loops and implementing native would speed up data processing.

**Comment by dockimbel** [comment]
That feature already exists using `copy/part`:
```
>> table: [[a b c d] [x y z w] [5 6 7 8]]
== [[a b c d] [x y z w] [5 6 7 8]]
>> copy/part second table 1x3
== [x y]
```

**Comment by GiuseppeChillemi** [comment]
It seems we are not talking about the same feature:

My result is: 

`>>  [x z]` while your is `>>  [x y]`

My wish is to express the positions as single columns, so you can write:

```
table: [[a b c d] [x y z w] [5 6 7 8]]
pick second table [1 3 4]
```
And get the result:

`>>  [x z w]`

**Comment by greggirwin** [comment]
Do we have any docs or design notes on `copy` with `pair!` parts? I remember it being there, but its design isn't great IMO.

**Comment by dockimbel** [comment]
> It seems we are not talking about the same feature:

Indeed, I'm reopening it.
